handlers/api.go

```diff
@@ -1667,14 +1667,14 @@ func ApiValidatorWithdrawals(w http.ResponseWriter, r *http.Request) {
 
 	q := r.URL.Query()
 
-	latestEpoch := services.LatestEpoch()
 	epoch, err := strconv.ParseUint(q.Get("epoch"), 10, 64)
 	if err != nil {
-		epoch = latestEpoch
+		epoch = services.LatestEpoch()
 	}
 
-	endEpoch := epoch - 100
-	if epoch < 100 {
+	// startEpoch and endEpoch are both inclusive, so substracting 99 here will result in a limit of 100 epochs
+	endEpoch := epoch - 99
```

Comment:
**Minor:** Since endEpoch is created as uint64 it can get set to something like 18446744073709551527 for a line.
Maybe something like
```suggestion
endEpoch, err := math.Sub(epoch, 99)
if err != nil {
   endEpoch = 0
}
```
would be prefered?